sys/net/gnrc/routing/rpl/srh/gnrc_rpl_srh.c

```diff
@@ -41,7 +41,7 @@ int gnrc_rpl_srh_process(ipv6_hdr_t *ipv6, gnrc_rpl_srh_t *rh)
     uint8_t *addr_vec = (uint8_t *) (rh + 1);
     bool found = false;
 
-    DEBUG("RPL SRH: %" PRIu8 " addresses in the routing header\n", n);
+    DEBUG("RPL SRH: %u addresses in the routing header\n", n);
```

Comment:
is the cast really needed? `uint8_t` has probably always a lower type rank than `(unsigned)`? And casting to a higher rank of the same type was never necessary, or am I mistaken?